### PR TITLE
Remove SDF Code

### PR DESCRIPTION
**Pull request by PythonFZ**
Fix Issues with open3D e.g. in binder

- ZnVIS and open3D were either commented out or removed from the MDSuite code.
- SDF Calculator was removed from the calculators imported by default
- ZnVIS and open3d were removed from the `requirements.txt`

### CODE REVIEW

File: mdsuite/calculators/__init__.py [hunk 1]
```diff
@@ -190,8 +188,3 @@ def StructureFactor(self):
     def EinsteinHelfandThermalConductivity(self):
         """Calculator Property"""
         return EinsteinHelfandThermalConductivity(**self.kwargs)
-
-    @property
```
Comment (SamTov) [inline mdsuite/calculators/__init__.py]: I would prefer these are just commented out. They will return soon.

File: mdsuite/experiment/experiment.py [hunk 2]
```diff
@@ -364,88 +362,6 @@ def perform_transformation(self, transformation_name, **kwargs):
         transformation_run = transformation(self, **kwargs)
         transformation_run.run_transformation()  # perform the transformation
 
-    def run_visualization(
```
Comment (SamTov) [inline mdsuite/experiment/experiment.py:365]: Same as above

File: mdsuite/experiment/run.py [hunk 3]
```diff
@@ -235,7 +234,3 @@ def StructureFactor(self) -> StructureFactor:
     @property
     def EinsteinHelfandThermalConductivity(self) -> EinsteinHelfandThermalConductivity:
         return self.exp_wrapper(EinsteinHelfandThermalConductivity)(**self.kwargs)
-
-    @property
```
Comment (SamTov) [inline mdsuite/experiment/run.py]: Same as above

**Review by SamTov — CHANGES_REQUESTED** [review]
Just some comments about commenting things out rather than removing them.

File: mdsuite/calculators/__init__.py (hunk 1 above)
Comment (PythonFZ) [reply mdsuite/calculators/__init__.py]: Good point

File: mdsuite/experiment/run.py (hunk 3 above)
Comment (PythonFZ) [reply mdsuite/experiment/run.py]: resolved in fc233f4

File: mdsuite/experiment/experiment.py (hunk 2 above)
Comment (PythonFZ) [reply mdsuite/experiment/experiment.py:365]: resolved in fc233f4

File: mdsuite/calculators/__init__.py (hunk 1 above)
Comment (PythonFZ) [reply mdsuite/calculators/__init__.py]: resolved in fc233f4

**Review by SamTov — APPROVED** [review]
Looks good.

**Pull request merged**